db/seeds.rb

```diff
@@ -1,7 +1,49 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require 'faker'
+EMAIL = %w(trouni@le.com saad@le.com eugene@le.com alex@le.com)
+puts 'deleting boockings'
+Booking.destroy_all
+puts 'deleting trips'
+Trip.destroy_all
+puts 'deleting users'
+User.destroy_all
+puts 'deleting reviews'
+# Review.destroy_all
+puts 'deleting Review...'
+i = 0
+4.times do
```

Comment:
A recommendation for your iterations: you rarely need to use i = 0, i += 1 in your loops. Using ruby enumerators is simpler and more readable. For example, instead of the 4.times do, you can use each on your EMAIL array (which will still work even if you add more emails to your array).
```suggestion
EMAIL.each do |email|
  user = User.create!( email: email, password: 'secret' )
  # the rest of your code here
end
```